Formula/linuxmain-generator.rb

```diff
@@ -0,0 +1,15 @@
+class LinuxmainGenerator < Formula
+  desc "A shell command to automatically add code to Swift Package Manager projects to run unit tests on Linux."
+  homepage "https://github.com/valeriomazzeo/linuxmain-generator"
+  url "https://github.com/valeriomazzeo/linuxmain-generator/archive/0.2.0.tar.gz"
+  sha256 "84a7a29ef8680426d52591631610b683c853910c49532ad17d7101bd35ed207b"
+
+  def install
+    system "unset CC; swift build -c release"
```

Comment:
Why do we need to unset `CC` here? Also, please break it into something like:
```rb
ENV["CC"] = ""
system "swift", "build", "-c", "release"
```

If it only builds a binary you can even do `system "swift", "build", "-c", bin.to_s` and remove the `bin.install` below.